tests/app/dao/test_inbound_numbers_dao.py

```diff
@@ -33,12 +33,14 @@ def test_get_inbound_numbers(db_session):
 class TestGetAvailableInboundNumbers:
 
     def test_gets_available_inbound_number(self, notify_db, notify_db_session):
-        inbound_number = create_inbound_number(number='1')
+        inbound_number = create_inbound_number(number='1', url_endpoint="https://example.foo", self_managed=True)
 
         res = dao_get_available_inbound_numbers()
 
         assert len(res) == 1
         assert res[0] == inbound_number
+        assert hasattr(res[0], "url_endpoint")
+        assert hasattr(res[0], "self_managed")
```

Comment:
I think we should just double check it accepts a "True" setting for self-managed after ensuring it has the attribute.

Reply:
In addition to line 41?  If the assertions on line 41 and 43 are valid, self_managed must be True.

Reply:
I added more conditions.

Reply:
I agree it should be True. Tests are where we assert the truth. 
Maybe I'm being too picky; I'll concede to a second opinion from @cris-oddball

Reply:
I'm weighing in too late since dave updated this, but I agree, assert for true.

Reply:
done